### DISCUSSION TITLE
kairship/workspace

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2355] [Announcements]
# kairship/workspace

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/workspace.html

**Comment by yt-huang** [comment]
希望增加对多云命名空间的介绍

**Comment by ModetaNiu** [comment]
已新增页面， See：https://docs.daocloud.io/kairship/resource/ns.html
